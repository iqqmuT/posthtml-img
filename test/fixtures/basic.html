<p><img src="https://www.kernel.org/theme/images/logos/tux.png"></p>
