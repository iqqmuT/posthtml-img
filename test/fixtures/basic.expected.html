<p><img src="https://www.kernel.org/theme/images/logos/tux.png" width="104" height="120"></p>
